pydata_sphinx_theme/static/css/theme.css

```diff
@@ -0,0 +1,85 @@
+/*****************************************************************************
+ * Custom font
+**/
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+:root {
+  /*****************************************************************************
+   * Bootstrap variables are availlable:
+   * https://getbootstrap.com/docs/4.0/getting-started/theming/#css-variables
+  **/
+
+  /*****************************************************************************
+  * Font size
+  **/
+  --fs-base: 15px; /* base font size - applied at body / html level */
+
+  /* heading font sizes */
+  --fs-h1: 36px;
```

Comment:
Should we make those relative (`em`) sizes?

Reply:
we could make those relative, but since we allow this to be defined in a 'setting', defining via `em` is harder to think about. Since 1em == `--fs-base`. Since mostly no inheritance happend on heading level, defining in pixels or ems is little difference I think.